artifactory_cleanup/rules/base.py

```diff
@@ -293,21 +294,32 @@ def filter(self, artifacts: ArtifactsList) -> ArtifactsList:
                 print()
         return artifacts
 
-    def delete(self, artifact: ArtifactDict, destroy: bool) -> None:
+    def delete(self, artifact: ArtifactDict, destroy: bool, ignore_not_found: bool) -> None:
```

Comment:
Also let's define the default behavior as it's right now
```suggestion
    def delete(self, artifact: ArtifactDict, destroy: bool, ignore_not_found: bool = False) -> None:
```